framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -56,9 +68,11 @@ player_result_t MediaPlayerImpl::create()
 
 void MediaPlayerImpl::createPlayer(player_result_t &ret)
 {
-	medvdbg("MediaPlayer Worker : createPlayer\n");
+	LOG_STATE_INFO(mCurState);
+
 	if (mCurState != PLAYER_STATE_NONE) {
-		meddbg("MediaPlayer create fail : wrong state\n");
+		meddbg("%s Fail : invalid state\n", __func__);
+		LOG_STATE_DEBUG(mCurState);
```

Comment:
in this case, 
line 71 : LOG_STATE_INFO(mCurState); 
and
line 75 : LOG_STATE_DEBUG(mCurState);
there are the same result.

i think doesn't need this debug message.
how about think?
or how about setup for info or debug in kconfig.